core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -120,7 +120,8 @@ class KafkaApis(val requestChannel: RequestChannel,
                 time: Time,
                 val tokenManager: DelegationTokenManager,
                 val apiVersionManager: ApiVersionManager,
-                val clientMetricsManager: Option[ClientMetricsManager]
+                val clientMetricsManager: Option[ClientMetricsManager],
+                val defaultActionQueue: ActionQueue
```

Comment:
I thought by adding ActionQueue in KafkaApis, we could get rid of ReplicaManager.tryCompleteActions. But we can't because of the dependency from DelayedJoin. So, it's better to not add ActionQueue here. Could we revert that logic? Sorry for the confusion.